core/frontend/src/tile/map/ImageryTileTree.ts

```diff
@@ -142,6 +151,35 @@ export class ImageryMapTile extends RealityTile {
   }
 }
 
+/** Object that holds various state values for a MapTileTree

```

Comment:
Not various, only one - a scale range visibility. Is a new class+object necessary?

Reply:
Since we keep the state in MapTileTree._imageryTreeState, I felt creating state object would be more flexible in the long run compared to a dedicated map<Id64String, MapTileTreeScaleRangeVisibility>.

Reply:
discussed below.